### PR TITLE
[onert/cpu] Support FusedBatchNorm custom operation

### PR DESCRIPTION
**Pull request by KimDongEon**
Implement FusedBatchNorm custom operation for cpu backend

### CODE REVIEW

File: runtime/onert/core/src/compiler/OperationValidator.cc [hunk 1]
```diff
@@ -519,6 +519,21 @@ void OperationValidator::visit(const ir::operation::Floor &node)
   OP_REQUIRES(_ctx.at(output_index).shape() == _ctx.at(input_index).shape());
 }
 
+void OperationValidator::visit(const ir::operation::FusedBatchNorm &node)
```
Comment (hseok-oh) [inline runtime/onert/core/src/compiler/OperationValidator.cc:522]: You don't need to implement `OperationValidator` if there is nothing to check

File: runtime/onert/core/include/ir/operation/FusedBatchNorm.h [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+public:
+  enum Input
+  {
+    INPUT_X = 0,
+    INPUT_SCALE = 1,
+    INPUT_OFFSET = 2,
+    INPUT_EPSILON = 3,
+    INPUT_MEAN = 4,
+    INPUT_VARIANCE = 5
+  };
```
Comment (hseok-oh) [inline runtime/onert/core/include/ir/operation/FusedBatchNorm.h:40]: As I know, our internal tflite model file's `CUSTOM(FusedBatchNorm)` has 5 inputs and 6 outputs. Could you tell us where do can we find this input/output size and role?

File: runtime/onert/core/include/ir/operation/FusedBatchNorm.h [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  enum Output
+  {
+    OUTPUT_Y = 0,
+    OUTPUT_BATCH_MEAN = 1,
+    OUTPUT_BATCH_VARIANCE = 2
+  };
+
+public:
+  FusedBatchNorm(const OperandIndexSequence &inputs, const OperandIndexSequence &outputs);
+
```
Comment (hseok-oh) [inline runtime/onert/core/include/ir/operation/FusedBatchNorm.h:51]: Does `CUSTOM(FusedBatchNorm)` have no parameter?

File: runtime/onert/frontend/nnapi/wrapper/OperationFactory.cc [hunk 4]
```diff
@@ -2079,6 +2079,24 @@ OperationFactory::OperationFactory()
   };
 }
 
+_map[ANEURALNETWORKS_FUSEBATCHNORM_EX] = [](const OperationFactory::Param &init_param, Operands &) {
+    assert(init_param.input_count == 3 && init_param.output_count == 1);
```
Comment (hseok-oh) [inline runtime/onert/frontend/nnapi/wrapper/OperationFactory.cc:2083]: `assert` condition does not meet below comment.

File: runtime/onert/frontend/base_loader/include/base_loader.h [hunk 5]
```diff
@@ -1702,6 +1715,9 @@ void BaseLoader<LoaderDomain, SpecificLoader>::loadOperation(const Operator *op,
     case BuiltinOperator::BuiltinOperator_TILE:
       loadTile(op, subg);
       return;
+    case BuiltinOperator::BuiltinOperator_loadFusedBatchNorm:
```
Comment (hseok-oh) [inline runtime/onert/frontend/base_loader/include/base_loader.h:1718]: We will not use tflite and circle builtin operation for `FusedBatchNorm` operation.

**Comment by hseok-oh** [comment]
@KimDongEon Please check my comment. #1449 can be reference for implementation.

File: runtime/onert/core/include/ir/operation/FusedBatchNorm.h (hunk 3 above)
Comment (KimDongEon) [reply runtime/onert/core/include/ir/operation/FusedBatchNorm.h:51]: tflite operation can be found parameter in tflite schema.fbs. But I don't know how to find custom operation.

Is there a way to find it?

File: runtime/onert/core/include/ir/operation/FusedBatchNorm.h (hunk 2 above)
Comment (KimDongEon) [reply runtime/onert/core/include/ir/operation/FusedBatchNorm.h:40]: I will revised input/output to 5/6 by referring to the official document of tensorflow.

File: runtime/onert/core/src/compiler/OperationValidator.cc (hunk 1 above)
Comment (KimDongEon) [reply runtime/onert/core/src/compiler/OperationValidator.cc:522]: How should I determine if I need to check?
I don't know the standard.

File: runtime/onert/frontend/nnapi/wrapper/OperationFactory.cc (hunk 4 above)
Comment (KimDongEon) [reply runtime/onert/frontend/nnapi/wrapper/OperationFactory.cc:2083]: I will modify

File: runtime/onert/frontend/base_loader/include/base_loader.h (hunk 5 above)
Comment (KimDongEon) [reply runtime/onert/frontend/base_loader/include/base_loader.h:1718]: I will modify to call through loadCustom() referring to #1449.

**Comment by KimDongEon** [comment]
@hseok-oh 
Thank you for your opinion.

I have additional questions.

I have difficulty implementing cker/operation/FusedBatchNorm.h, and I have a question about it.

I am proceeding with reference to the link below.
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/kernels/fused_batch_norm_op.cc

I think 283-326 lines are implementation.

Here
How should tensorflow::CPUDevice be converted to

File: runtime/onert/core/src/compiler/OperationValidator.cc (hunk 1 above)
Comment (hseok-oh) [reply runtime/onert/core/src/compiler/OperationValidator.cc:510]: There is no standard because this is different on each operation. Maybe type checking?

**Comment by hseok-oh** [comment]
>How should tensorflow::CPUDevice be converted to

There is no simple way to convert tensorflow kernel code. It's better to understand tensorflow kernel code (operation) and implement in `cker` yourself.

**Pull request closed without merging**